lib/ansible/modules/windows/win_get_url.py

```diff
@@ -31,23 +31,21 @@
 version_added: "1.7"
 short_description: Fetches a file from a given URL
 description:
- - Fetches a file from a URL and saves to locally
- - For non-Windows targets, use the M(get_url) module instead.
+  - Fetches a file from a URL and saves to locally.
+  - For non-Windows targets, use the M(get_url) module instead.
 author:
-    - "Paul Durivage (@angstwad)"
-    - "Takeshi Kuramochi (tksarah)"
+  - Paul Durivage (@angstwad)
+  - Takeshi Kuramochi (tksarah)
 options:
   url:
     description:
       - The full URL of a file to download
-    required: true
-    default: null
+    required: yes
   dest:
     description:
       - The absolute path of the location to save the file at the URL. Be sure
```

Comment:
This now works without it being an absolute path, we would want to document that.